### PR TITLE
Add new create meeting param allowPromoteGuestToModerator

### PR DESCRIPTION
**Pull request by SamuelWei**
Implements #202

### CODE REVIEW

File: tests/unit/Parameters/CreateMeetingParametersTest.php [hunk 1]
```diff
@@ -84,7 +85,6 @@ public function testCreateMeetingParameters()
         $this->assertEquals($params['meetingLayout'], $createMeetingParams->getMeetingLayout());
         $this->assertEquals($params['learningDashboardEnabled'], $createMeetingParams->isLearningDashboardEnabled());
         $this->assertEquals($params['learningDashboardCleanupDelayInMinutes'], $createMeetingParams->getLearningDashboardCleanupDelayInMinutes());
-        $this->assertEquals($params['allowModsToEjectCameras'], $createMeetingParams->isAllowModsToEjectCameras());
```
Comment (FelixJacobi) [inline tests/unit/Parameters/CreateMeetingParametersTest.php:87]: Why is that removed?

**Review by FelixJacobi — CHANGES_REQUESTED** [review]

File: tests/unit/Parameters/CreateMeetingParametersTest.php (hunk 1 above)
Comment (SamuelWei) [reply tests/unit/Parameters/CreateMeetingParametersTest.php:87]: It is the same as Line 65

Comment (FelixJacobi) [reply tests/unit/Parameters/CreateMeetingParametersTest.php:87]: Ok, missed that.

**Review by FelixJacobi — APPROVED** [review]
Coveralls must be fixed.
